### ISSUE TITLE
Include SmartGadget Application in the official F-Droid Repository

### ISSUE THREAD

**Issue by honggoff** [Sensirion/SmartGadget-Android#30]
Since this is a true open-source application (from what I can tell, all dependencies are open source as well), it would be a shame not to include it in the F-Droid repository.

I will look into this and keep this issue updated.

**Comment by honggoff** [comment]
See pull request on gitlab: https://gitlab.com/fdroid/fdroiddata/merge_requests/1715

**Comment by abrauchli** [comment]
:+1: Awesome, seems merged already. I'll leave it open until it's downloadable from within Fdroid .. or you can close it, whichever comes first.

**Comment by honggoff** [comment]
And there it is in the repository: https://f-droid.org/repository/browse/?fdfilter=smart+gadget&fdid=com.sensirion.smartgadget
I tested it and it works on my device.

I went through the permissions it requires, and was surprised to find WRITE_EXTERNAL_STORAGE and READ_PHONE_STATE. It turns out, these are implicit permissions from androidplot, because it has a targetSdkVersion smaller than 4: https://developer.android.com/studio/build/manifest-merge.html#implicit-permissions
I think it would be nice to get rid of these.
